imagetest/fixtures.go

```diff
@@ -234,13 +241,19 @@ func (t *TestVM) SetCustomNetwork(network *Network, subnetwork *Subnetwork) erro
 	networkInterface := compute.NetworkInterface{
 		Network:    network.name,
 		Subnetwork: subnetworkName,
+		NetworkIP:  networkIP,
+
 		AccessConfigs: []*compute.AccessConfig{
 			{
 				Type: "ONE_TO_ONE_NAT",
 			},
 		},
 	}
-	t.instance.NetworkInterfaces = []*compute.NetworkInterface{&networkInterface}
+	if t.instance.NetworkInterfaces == nil {
+		t.instance.NetworkInterfaces = []*compute.NetworkInterface{&networkInterface}
+	} else {
+		t.instance.NetworkInterfaces = append(t.instance.NetworkInterfaces, &networkInterface)
+	}
 
```

Comment:
Based on the function's name and documentation, I would have expected any current network interfaces to be replaced. (The function talks about "set" rather than "append")

Reply:
The function handle two cases. create 1st interface if not exist or append one more.

Reply:
The word `set` typically implies overwrite. So if you want the described behavior, you'd want to name the function `AddCustomNetwork` or `AppendCustomNetwork`

Reply:
Agree